RecordBuffers/src/main/java/datamine/storage/recordbuffers/ReadOnlyRecord.java

```diff
@@ -308,14 +309,14 @@ private void constructOffsetArrayFromRecordBuffer() {
 					case STRING:
 						// string requires a SHORT to store its length (max < 32k)
 						short arrayLength = bytebuffer.getShort(offset);
-						if (arrayLength > 0) {
+						if (arrayLength >= 0) {
```

Comment:
Why adding "=" here? Are we going to add 2 to offset even if there is no short value in bytebuffer?